### DISCUSSION TITLE
Bare code being displayed when subscribing to newsfeed

### DISCUSSION THREAD

**Discussion by Korabi-dev** [community/community#21434] [Other Feature Feedback, Questions, & Ideas]
<p>When I tried to subscribe to a newsfeed of an organization I was in, it just showed me this<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/9/29e7ef311c353443269e28b2a084c8bbf1966e68.png" data-download-href="/uploads/short-url/5YIuAlA8yehYrJiPy0JJ11Kk4D6.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/69105206/181064534-e8fbc2ef-5008-4112-95f8-7452de623800.png" alt="image" data-base62-sha1="5YIuAlA8yehYrJiPy0JJ11Kk4D6" width="690" height="351" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/9/29e7ef311c353443269e28b2a084c8bbf1966e68_2_690x351.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/9/29e7ef311c353443269e28b2a084c8bbf1966e68_2_1035x526.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/2/9/29e7ef311c353443269e28b2a084c8bbf1966e68_2_1380x702.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/9/29e7ef311c353443269e28b2a084c8bbf1966e68_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×979 63.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Milo123459** [comment, marked as answer]
<p>this isnt a bug, this is the RSS file</p>

**Comment by Korabi-dev** [comment]
<p>milo you told me on discord already</p>
